domain/src/main/java/org/apache/rocketmq/eventbridge/domain/model/quota/QuotaService.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.rocketmq.eventbridge.domain.model.quota;
+
+public interface QuotaService {
+    /**
+     * Return the api destination total quota.
+     * @param accountId
+     * @return
+     */
+    Integer getApiDestinationTotalQuota(String accountId);
```

Comment:
Use a common API, and add a parameter to specify different templates

Reply:
Modified to use common API